project_timeline_critical_path/tests/test_critical_path.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestCriticalPath(TransactionCase):
+    def test_diff_project_critical_path(self):
+        project_obj = self.env['project.project']
+        project1 = project_obj.create({
+            'name': 'Test1'
+        })
+        project2 = project_obj.create({
+            'name': 'Test2'
```

Comment:
Tests are failing because of the way module `project_key`  generates keys for a project.
The generated keys for 'Test1' and 'Test2' are both equal to 'TES', and that is causing the issue because of the uniqueness constraint on the keys.

I you give a different name to this project, so for example instead of 'Test2' you name it something like 'Other Test', the tests will pass.